test/_assert.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+var assert = require('assert');
+
+/**
+ * Tests equality of Date objects, w/o considering milliseconds.
+ * @see {@link https://github.com/joyent/node/issues/7000|File timestamp resolution is inconsistent with fs.stat / fs.utimes}
+ */
+exports.assertDateEqual = function (actual, expected, message) {
+	actual = new Date(actual);
+	expected = new Date(expected);
+
+	actual.setMilliseconds(0);
+	expected.setMilliseconds(0);
+
+	assert.equal(actual.getTime(), expected.getTime(), message);
+};
```

Comment:
This should no longer be needed now that we've dropped support for 0.10 and 0.12.